### PR TITLE
Updated the FontAwesomeIcons with CareIcons, replaced PageTitle with Page component and migrated tooltip.

### LINKED ISSUE #4957: Replace Material UI components in `src/Components/Facility/InventoryLog.tsx`:
```tsx
13: import { Tooltip } from "@material-ui/core";
```

### Tasks

Ensure all the below tasks are accomplished if applicable to the file being modified in the PR.

- [ ] Migrate all Material UI components to use new Care UI / custom components. (ref: [🛠️ Tailwinding Care](https://github.com/coronasafe/care_fe/issues/3742))
- [ ] Migrate all Material UI icons to use [🏥 **Care Icons**](https://github.com/coronasafe/care_fe/pull/4071).
- [ ] Migrate `PageTitle` to `Page` component if any. (See [example](https://github.com/coronasafe/care_fe/pull/4796/files#diff-8e39f154ccf13c9dedd0f968d21a0abbde450e88df9724a4e7d4264d487a7846))

For more information see the epic issue that tracks this issue.

### PR DESCRIPTION
**Pull request by yogeshbhutkar**
### <samp>🤖 Generated by Copilot at ee3e999</samp>

Refactored the InventoryLog component to improve the UI and fix a bug. Used new components from the UI library and SVG icons.

## Proposed Changes

- Fixes #4957 
- Migrated the tooltip.
- Migrated the PageTitle with Page Component.
- Added Care's icons.

An image for the same has been attached for the reference below.

image

### <samp>🤖 Generated by Copilot at ee3e999</samp>

*  Refactor the InventoryLog component to use the Page and CareIcon components from the UI library and fix some UI bugs ([link](https://github.com/coronasafe/care_fe/pull/5642/files?diff=unified&w=0#diff-663ba30da5cdcaeb8b4b10df234a346423c5818601577a4633ec6920e57fae97L13-R15), [link](https://github.com/coronasafe/care_fe/pull/5642/files?diff=unified&w=0#diff-663ba30da5cdcaeb8b4b10df234a346423c5818601577a4633ec6920e57fae97L149-R151), [link](https://github.com/coronasafe/care_fe/pull/5642/files?diff=unified&w=0#diff-663ba30da5cdcaeb8b4b10df234a346423c5818601577a4633ec6920e57fae97L158-R158), [link](https://github.com/coronasafe/care_fe/pull/5642/files?diff=unified&w=0#diff-663ba30da5cdcaeb8b4b10df234a346423c5818601577a4633ec6920e57fae97L165-R166), [link](https://github.com/coronasafe/care_fe/pull/5642/files?diff=unified&w=0#diff-663ba30da5cdcaeb8b4b10df234a346423c5818601577a4633ec6920e57fae97L176-R184), [link](https://github.com/coronasafe/care_fe/pull/5642/files?diff=unified&w=0#diff-663ba30da5cdcaeb8b4b10df234a346423c5818601577a4633ec6920e57fae97L248-R247), [link](https://github.com/coronasafe/care_fe/pull/5642/files?diff=unified&w=0#diff-663ba30da5cdcaeb8b4b10df234a346423c5818601577a4633ec6920e57fae97L256-R285))

### CODE REVIEW

**Review by rithviknishad — CHANGES_REQUESTED** [review]
You haven't replaced html buttons with ButtonV2. Other than that everything LGTM

**Comment by yogeshbhutkar** [comment]
I have made the above mentioned changes and have requested for a review. The component is working as expected and the issue now seems to be ready to close.

File: src/Components/Facility/InventoryLog.tsx [hunk 1]
```diff
@@ -146,43 +147,42 @@ export default function InventoryLog(props: any) {
...
+              )}
+            </div>
+            <ButtonV2
               onClick={(_) => flagFacility(inventoryItem.external_id)}
               disabled={saving}
-              className="btn btn-default"
+              variant="secondary"
             >
               {inventoryItem.probable_accident ? (
                 <span className="text-primary-500">
```
Comment (rithviknishad) [inline src/Components/Facility/InventoryLog.tsx]: Avoid setting the color of text inside ButtonV2. ButtonV2 applies it based on the variant.

This is to keep design consistent and easy to maintain.

See how other Buttons with icons are used.

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Review by rithviknishad — APPROVED** [review]
You could have simply added a ternary operator in the variant instead of the entire component. But anyways, it's fine.

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**